controllers/fenceagentsremediation_controller_test.go

```diff
@@ -184,7 +185,7 @@ var _ = Describe("FAR Controller", func() {
 
 				By("Not having finalizer")
 				farNamespacedName.Name = underTestFAR.Name
-				Eventually(func(g Gomega) bool {
+				Consistently(func(g Gomega) bool {
```

Comment:
I'm wondering how useful Consitently is with a 100ms duration only 🤔

Reply:
Before the change eventually ran only once, 10 ms, and now it waits longer which helps underTestFAR in [line 204](https://github.com/medik8s/fence-agents-remediation/pull/74/files#diff-fac0cf56dda42683c1e24b46c87cfa4cca46c444145ee69de9e264db50f10289R204) to be updated with the last status update, so we can check that `LastUpdateTime` is not nil

Reply:
sorry, but that's the wrong approach:

- with eventually you wait until something happens
- with consistently you ensure that something does NOT change

So here you want to ensure that no finalizer is added. For that using Consistently is correct, but you did it for the wrong reason 😉 That low duration though does add that much value to the test IMHO. But that's a nit.

When you have a issue in line 204, fix it there, not here. Use Eventually over there if you need to wait.